portal-backend/cron/hemi-supply/src/config.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const env = (variable, defaultValue) => process.env[variable] || defaultValue
+
+const config = {
+  contracts: {
+    hemi: {
+      bnb: '0x5ffd0eadc186af9512542d0d5e5eafc65d5afc5b',
+      eth: '0xEb964A1A6fAB73b8c72A0D15c7337fA4804F484d',
+      hemi: '0x99e3dE3817F6081B2568208337ef83295b7f591D',
+    },
+    merkleBox: '0x9Ab3660ceE733332785cEa09D1a4Ff222F31aE54',
+    safe: '0x694fA0816999Da16E8783C0f5cDE68c13a33C4e6',
+    veHemi: '0x371d3718D5b7F75EAb050FAe6Da7DF3092031c89',
```

Comment:
not for this PR, but I'm starting to wonder if we should move this also into a package of "known addresses", similar to the token list..

Reply:
Good point. We will probably end up doing so.